src/components/organisms/CategoryList/index.tsx

```diff
@@ -63,15 +62,15 @@ function CategoryList({ reRenderCategory, selectedCategory, categoryChange }: IP
           </div>
         </div>
         <div className="shadow"></div>
-        <img className="next" src={nextIcon} onClick={nextSlide} alt="" />
-        <img src={moreClicked ? moreClickedIcon : moreIcon} onClick={onClickOpenMore} alt="" />
+        <img className="next" src={NextIcon} onClick={nextSlide} alt="" />
+        <img src={moreClicked ? MoreClickedIcon : MoreIcon} onClick={onClickOpenMore} alt="" />
```

Comment:
```suggestion
        <img src={isMoreClicked ? MoreClickedIcon : MoreIcon} onClick={onClickOpenMore} alt="" />
```
혹시 true/false로 하는값은 is가 붙어도 될까요??